lib/Textmaster/Model/Project.php

```diff
@@ -207,6 +224,8 @@ public function setCallback(array $callback)
         }
 
         $this->data['callback'] = $callback;
+
+        return $this;
```

Comment:
Yes, in a [unit test](https://github.com/worldia/textmaster-api/blob/master/test/Textmaster/Unit/Model/ProjectTest.php#L97).